mobbex-for-woocommerce.php

```diff
@@ -164,10 +179,10 @@ public function plugin_row_meta($links, $file)
     {
         if (strpos($file, plugin_basename(__FILE__)) !== false) {
             $plugin_links = [
-                '<a href="' . esc_url(MobbexGateway::$site_url) . '" target="_blank">' . __('Website', MOBBEX_WC_TEXT_DOMAIN) . '</a>',
-                '<a href="' . esc_url(MobbexGateway::$doc_url) . '" target="_blank">' . __('Documentation', MOBBEX_WC_TEXT_DOMAIN) . '</a>',
-                '<a href="' . esc_url(MobbexGateway::$github_url) . '" target="_blank">' . __('Contribute', MOBBEX_WC_TEXT_DOMAIN) . '</a>',
-                '<a href="' . esc_url(MobbexGateway::$github_issues_url) . '" target="_blank">' . __('Report Issues', MOBBEX_WC_TEXT_DOMAIN) . '</a>',
+                '<a href="' . esc_url(MobbexGateway::$site_url) . '" target="_blank">' . __('Website', 'woocommerce-mobbex-gateway') . '</a>',
```

Comment:
El text domain es una referencia para que wordpress encuentre las traducciones, habría que dejarlo en mobbex-for-woocommerce para que estas funcionen (o también se puede usar la constante MOBBEX_WC_TEXT_DOMAIN, pero tiene varias contras también)